tests/conftest.py

```diff
@@ -104,6 +104,12 @@ def _get_factory(package, factory_name):
     return _get_factory
 
 
+@pytest.fixture
+def owned_contract():
+    with open(V2_PACKAGES_DIR / "owned" / "contracts" / "Owned.sol") as file_obj:
```

Comment:
I think you can also do `(V2_PACKAGES_DIR / "owned" / "contracts" / "Owned.sol").open()` but I'm undecided on whether I think it's a good pattern.  The outer parenthesis wrapping is sort of awkward.  Not advocating change, just bringing up my thoughts.

Reply:
Agreed on the parenthesis wrapping being weird, but i'm a fan of pathlib and think the weirdness is worth it